### DISCUSSION TITLE
rev 2 solder jumper question

### DISCUSSION THREAD

**Discussion by doom-cat** [siderakb/ergo-snm-keyboard#13] [Q&A] [answered]
I am new to the world of custom keyboards, but I really like this design, so I am trying to order the PCBs to build a rev 2. (I do not need wireless, so I thought I would start with a slightly simpler build.) However, I hit a snag when I tried to upload the PCL file for the right PCB. The gerber and BOM files include solder jumpers JP1 and JP2, but the PCL file does not. This seems to be an issue for JLCPCB, and I am not sure how to proceed. I see that the jumpers are not included in the rev 3 version. Are they needed in rev 2?

Once again, I am new to the hobby, so I don't really know what I'm doing, but I am eager to learn. Thank you so much.

**Comment by ziteh** [comment, marked as answer]
Sorry for the late reply
JP1 and JP2 are not an actual part, so they do not exist in the CPL file and can be safely ignored (i.e. do not place the part)

**Comment by ziteh** [comment]
Close inactive issue, and please feel free to reopen it if needed : )
